app/javascript/src/components/Dashboard/Contacts/constants.js

```diff
@@ -10,20 +11,13 @@ const DUMMY_CONTACT_DATA = {
 
 export const ROW_DATA = Array(100).fill(DUMMY_CONTACT_DATA);
 
-export const ROLES = [
-  {
-    label: "UX designer",
-    value: "uxDesigner",
-  },
-  {
-    label: "Tester",
-    value: "tester",
-  },
-  {
-    label: "Analyst",
-    value: "analyst",
-  },
-];
+export const ROLES = buildSelectOption([
```

Comment:
You haven't gone through my suggestion very well. Here we are building `options`. Not a single `option`. Thus plural. Not singular.

So function name should be `buildSelectOptions`.

Reply:
Apologies for the oversight. Will fix ASAP